### PR TITLE
UIIN-2180: View Instance record | Contributors accordion | Show authorized indicator for each contributor

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
## Purpose
Add a link to the MARC authority app when an `Instance` record with source = `MARC` is linked to an authority record. 

## Issues
[UIIN-2180](https://issues.folio.org/browse/UIIN-2180)

## Screenshots
image

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by mkuklis — APPROVED** [review]

File: src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js [hunk 1]
```diff
@@ -72,6 +77,46 @@ const InstanceContributorsView = ({
     );
   }, [contributors, contributorTypes, contributorNameTypes]);
 
+  const getName = (item) => {
+    if (segment === segments.instances && source === 'MARC' && 'item.authorityId') {
```
Comment (BogdanDenis) [inline src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js]: I think `item.authorityId` shouldn't be a string here, right?

File: src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js [hunk 2]
```diff
@@ -72,6 +77,46 @@ const InstanceContributorsView = ({
...
+      return (
+        <>
```
Comment (BogdanDenis) [inline src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js:85]: Can we make this element a separate component? It's just mostly a duplicate of https://github.com/folio-org/ui-inventory/pull/1794/files

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [reply src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js]: Thanks, fixed

File: src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js (hunk 2 above)
Comment (BogdanDenis) [reply src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js:85]: Resolving the comment - DOM structure inside InstancesList doesn't allow for this same component

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**